src/main/java/com/github/manolo8/darkbot/extensions/plugins/Plugin.java

```diff
@@ -33,10 +41,26 @@ public void setDefinition(PluginDefinition definition) {
         info = ConfigEntity.INSTANCE.getPluginInfo(definition);
     }
 
+    public void setUpdateDefinition(PluginDefinition definition) {
+        this.updateDefinition = definition;
+    }
+
+    public void setUpdateStatus(UpdateStatus status) {
+        updateStatus = status;
+    }
+
+    public void add(PluginIssue issue) {
+        updateIssues.getIssues().add(issue);
+    }
+
```

Comment:
```suggestion
```
This is a no-go, since when is "adding" to a plugin is intuitive for "adding to the update issues" ? It'd be very confusing for any other develiper, just use `plugin.getUpdateIssues().add(issue)` which is very explicit about what it's doing